main/BreakfastPizza.md

```diff
@@ -0,0 +1,32 @@
+# Breakfast Pizza
+
+Servings: Large pie of da pizza  
+Prep time: 15 min.  
+Cook time: 15 min.  
+Total time 30 min.  
+
+## Equipment
+- Round pizza pan/sheet
+- Pizza cutting device
+
+## Ingredients
+- 1 frozen pizza dough - Gordon Food Service
+- Vegetable Oil
+- Cheddar Cheese Sauce - Sam's Club
+- Mozzarella Cheese
+- Cheddar Cheese
```

Comment:
How much of lines 14-17? Do I use 1 vegetable oil? :P

Reply:
I don't know, until it looks good basically. I didn't measure anything when I made it.

Reply:
A handful of Cheese Sauce, a pinch of oil lol